from .api_exception import ApiException


class NotFoundException(ApiException):
    pass
